src/main/steps/c100-rebuild/check-your-answers/util/miam.util.ts

```diff
@@ -90,12 +90,12 @@ export const MiamContentsForDomensticVoilence = UserCase => {
           ...DomesticAbuseCy().letterOfBeingVictim_subFields,
           ...DomesticAbuseCy().letterFromAuthority_subFields,
           ...GeneralContentCy(),
-          generalReasonTitle: GeneralContentEn().title,
```

Comment:
Removed EN related titles? Where are EN related translations?

Reply:
Enlish content are available at english block. in cy block it was written as english, for that reason some content of check your aswer  screen was not translating